src/BuildScriptGenerator/Node/VersionProviders/NodeSdkStorageVersionProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System.Net.Http;
+
+namespace Microsoft.Oryx.BuildScriptGenerator.Node
+{
+    internal class NodeSdkStorageVersionProvider : SdkStorageVersionProviderBase, INodeVersionProvider
+    {
+        private PlatformVersionInfo _platformVersionInfo;
+
+        public NodeSdkStorageVersionProvider(IEnvironment environment, IHttpClientFactory httpClientFactory)
+            : base(environment, httpClientFactory)
+        {
+        }
+
+        // To enable unit testing
+        public virtual PlatformVersionInfo GetVersionInfo()
+        {
+            if (_platformVersionInfo == null)
+            {
+                _platformVersionInfo = GetAvailableVersionsFromStorage(
+                    platformName: "nodejs",
```

Comment:
use toolnameconstant?

Reply:
toolname is `node` but we have `nodejs` folder. This has been a problem for a while now. I will file a cleanup item for this to make everything `node` instead